### PR TITLE
[ISSUE #4276] remove redundant cookie setting

### PR DESCRIPTION
**Pull request by RayayChung**
fix: #4276 
Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-plugin/shenyu-plugin-httpclient/src/main/java/org/apache/shenyu/plugin/httpclient/WebClientPlugin.java [hunk 1]
```diff
@@ -63,8 +63,7 @@ protected Mono<ClientResponse> doRequest(final ServerWebExchange exchange, final
                         .flatMap(bytes -> Mono.fromCallable(() -> Optional.ofNullable(bytes))).defaultIfEmpty(Optional.empty())
                         .flatMap(option -> {
                             final ClientResponse.Builder builder = ClientResponse.create(response.statusCode())
-                                    .headers(headers -> headers.addAll(response.headers().asHttpHeaders()))
-                                    .cookies(cookies -> cookies.addAll(response.cookies()));
```
Comment (yu199195) [inline shenyu-plugin/shenyu-plugin-httpclient/src/main/java/org/apache/shenyu/plugin/httpclient/WebClientPlugin.java:67]: why remove cookies setting？

Comment (RayayChung) [reply shenyu-plugin/shenyu-plugin-httpclient/src/main/java/org/apache/shenyu/plugin/httpclient/WebClientPlugin.java:67]: Setting cookies is just part of setting headers. So the previous version codes may lead setting cookies twice.

Comment (yu199195) [reply shenyu-plugin/shenyu-plugin-httpclient/src/main/java/org/apache/shenyu/plugin/httpclient/WebClientPlugin.java:67]: Is there a time when cookies is not passed from the header?

Comment (RayayChung) [reply shenyu-plugin/shenyu-plugin-httpclient/src/main/java/org/apache/shenyu/plugin/httpclient/WebClientPlugin.java:67]: the data of `response.cookies()` is the subset of `response.headers().asHttpHeaders()`, therefore setting headers is also setting cookies in the sam time. I have never met the case that cookies is not present in headers. Is this case exists?

**Review by yu199195 — APPROVED** [review]

**Pull request merged**